README.md

```diff
@@ -1 +1,1205 @@
-# balena-cli-masterclass
\ No newline at end of file
+balenaCLI Masterclass
+=====================
+
+**Masterclass Type:** Core
+**Maximum Expected Time To Complete:** 60 minutes
+
+# Introduction
+
+The balena Command Line Interface (balenaCLI) utility consists of a number
+of commands that allow a user to develop, deploy and manage balena applications
+and devices.
+
+Everything that can be achieved via the balenaCloud dashboard can also be
+achieved via the balenaCLI.
+
+In this masterclass, you will learn how to:
+
+* Login to your account
+* Push application code to a balena Application
+* Deploy locally built code to a balena Application
+* SSH into a balena device
+* Push and build applications on a device on the local network for fast
+    development and prototyping
+* Use private Docker registries for base images and services
+* Create secret files and build arguments for building service images
+
+If you have any questions about this masterclass as you proceed through it,
+or would like clarifications on any of the topics raised here, please do
+contact us on the [balena forums](https://forums.balena.io/) where we'll
+be delighted to answer your questions.
```

Comment:
(or open an issue in this repo, perhaps?)

Reply:
Done.